packages/ui/src/1_atoms/Button/Button.tsx

```diff
@@ -0,0 +1,9 @@
+import React, { FC } from 'react';
+
+type ButtonProps = {
```

Comment:
I know this is not very important for the PR but pls, add `primary`, `size` and `label` as properties because in underlines them in the story.